### PR TITLE
docs(schema): document using `every` and `sql` in `refreshKey` together

### PR DESCRIPTION
**Pull request by hassankhan**
Story details: https://app.clubhouse.io/cube-dev/story/882

### CODE REVIEW

File: docs/content/Caching/Using-Pre-Aggregations.md [hunk 1]
```diff
@@ -25,7 +25,29 @@ Refresh strategy can be customized by setting the
...
+      granularity: `month`,
+      refreshKey: {
+        every: `12 hour`,
+      },
+    },
+  },
+});
+```
+
+Or by providing a [`sql` property][ref-schema-ref-preaggs-refresh-key-sql]
```
Comment (keydunov) [inline docs/content/Caching/Using-Pre-Aggregations.md:49]: we need to confirm with core and then mention it here - 

**what is the default value of every when only `sql` is specified?**

I assume it is 1 hour, but worth to double check.

File: docs/content/Schema/Reference/pre-aggregations.md [hunk 2]
```diff
@@ -493,6 +493,30 @@ cube(`Orders`, {
 For possible `every` parameter values please refer to
 [`refreshKey`][ref-cube-refreshkey] documentation.
 
+You can also use `every` with `sql`:
+
+<!-- prettier-ignore-start -->
+[[warning | ]]
+| When using `every` and `sql` together, the value of `every` should match
```
Comment (keydunov) [inline docs/content/Schema/Reference/pre-aggregations.md:500]: Is it coming from the core team?

**Review by keydunov — COMMENTED** [review]
There are two important questions to resolve before merging. Highlighted them in the comments.

File: docs/content/Caching/Using-Pre-Aggregations.md (hunk 1 above)
Comment (hassankhan) [reply docs/content/Caching/Using-Pre-Aggregations.md:49]: I'll add a link to https://cube.dev/docs/caching#in-memory-cache-default-refresh-keys which defines the defaults

File: docs/content/Schema/Reference/pre-aggregations.md (hunk 2 above)
Comment (hassankhan) [reply docs/content/Schema/Reference/pre-aggregations.md:500]: That's what I understood from the task in Notion, will confirm with @ovr

File: docs/content/Caching/Using-Pre-Aggregations.md [hunk 3]
```diff
@@ -25,7 +25,30 @@ Refresh strategy can be customized by setting the
...
+instead, and leaving [`every` unchanged from its' default
```
Comment (keydunov) [inline docs/content/Caching/Using-Pre-Aggregations.md:50]: in-memory and pre-aggs have different default.

Comment (hassankhan) [reply docs/content/Caching/Using-Pre-Aggregations.md:50]: I'll confirm with @ovr and update the PR accordingly

**Comment by keydunov** [comment]
@hassankhan Can we add an explanation, so if used together, Cube will check `sql` condition on the cadence defined in every. And if SQL condition is satisfied will refresh the pre-aggregation.

**Review by keydunov — CHANGES_REQUESTED** [review]
Commented above.

**Review by keydunov — APPROVED** [review]
Looks good to merge.

**Pull request merged**